asc2018/vdr/electrical/WaterlooMidnightSun_ASC2018VDR_Electrical.tex

```diff
@@ -81,65 +79,54 @@
 \listoffigures % <-------------- uncomment for list of figures
 %\listoftables % <-------------- uncomment for list of tables
 
-\section{History}
-Midnight Sun was founded in 1988 at the University of Waterloo. The team has produced 11 solar-powered vehicles since its inception, numbered MSI through MSXI. MSX and its predecessors have been traditional Challenger class cars. MSXI was the team's first attempt at a Cruiser class vehicle, which ultimately suffered from design issues relating to its monocoque design. With MSXII, the team has regrouped and designed a new Cruiser vehicle from the gound up, focussing strongly on reliability, safety, and manufacturability.
+\section{Overview}
```

Comment:
Nit: newline after any `\section`, here and throughout.

Reply:
I think this is the only place where I don't have a newline?